ding/worker/learner/base_learner.py

```diff
@@ -432,8 +436,11 @@ def policy(self, _policy: 'Policy') -> None:  # noqa
             Policy variable monitor is set alongside with policy, because variables are determined by specific policy.
         """
         self._policy = _policy
-        if self._rank == 0:
-            self._monitor = get_simple_monitor_type(self._policy.monitor_vars())(TickTime(), expire=10)
+        # if self._rank == 0:
+        #     self._monitor = get_simple_monitor_type(self._policy.monitor_vars())(TickTime(), expire=10)
+        
+        self._monitor = get_simple_monitor_type(self._policy.monitor_vars())(TickTime(), expire=10)
```

Comment:
add an argument named `only_monitor_rank0` to control the logic, defaults to True

Reply:
done